src/metabase/query_processor/pivot.clj

```diff
@@ -96,82 +124,116 @@
         ;; bottom right corner [_ _ _ _] => 1111 => Group #15
         [[]]))))))
 
-(defn- add-grouping-field
+(mu/defn ^:private keep-breakouts-at-indecies :- ::lib.schema/query
```

Comment:
typo: `indices`, throughout this function.

Or perhaps better, the also-accepted spelling of `indexes`.